ivy/functional/ivy/experimental/losses.py

```diff
@@ -23,7 +24,7 @@ def log_poisson_loss(
     *,
     compute_full_loss: bool = False,
     axis: int = -1,
-    reduction: str = "none",
```

Comment:
kindly uncommit this change too

Reply:
this commit is about log_poission_loss..should i really change,these changes are made by someone..

Reply:
The PR shows that you have updated the value of `reduction` to "mean", the only changes that should be shown should be just for `smooth_l1_loss` kindly un-commit the change which is being showed :slightly_smiling_face: